threads-api/src/threads-api.ts

```diff
@@ -111,7 +111,9 @@ export type ThreadsAPIPublishOptions =
   | {
       text?: string;
       parentPostID?: string;
-    } & ({ url?: string } | { image?: string });
+    } & ({ url?: string } | { image?: string | ThreadsAPIImage });
+
+export type ThreadsAPIImage = { name: string; data: Buffer };
```

Comment:
Since `name` is totally useless in that case, I would like to rename that to `type`/`mimeType`/`ext`(I'll put a 👍 to here) or whatever, and define `ThreadsAPIImage` like this:

```suggestion
export type ThreadsAPIImage = (
  { ext: string; data: Buffer } |
  { path: string; }
);
```